### PR TITLE
bug/event_table

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

**Review by pckhoi — APPROVED** [review]
LGTM. Before merging this PR, please merge main into this branch to trigger the process data job again.

File: lib/events.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 APPEAL_RECEIVE = "appeal_receive"
 APPEAL_DISPOSITION = "appeal_disposition"
 
-UOF_INCIDENT = "uof_incident"
+UOF_INCIDENT = "uof_occur"
```
Comment (pckhoi) [inline lib/events.py:46]: We might need to communicate this change with Huy

Comment (ayyubibrahimi) [reply lib/events.py:46]: Got it. On it after I review the changes.

**Comment by ayyubibrahimi** [comment]
@pckhoi when I review the latest changes, there is no change in the `uof_uid` in the events table, yet, on my local, the `uof_uid` is present for the 2600 `uof_occur` events.

**Comment by pckhoi** [comment]
Let's wait for the job to finish first.

**Comment by ayyubibrahimi** [comment]
Looks like it failed from a connection timeout:

`Post "https://hub.wrgl.co/api/users/ipno/repos/data/receive-pack/": read tcp 10.0.4.37:56568->35.184.35.134:443: read: connection timed out`

**Comment by pckhoi** [comment]
My service is probably OOM-killed but let's run the job again to make sure.

**Comment by pckhoi** [comment]
Looks like the event table size increased massively after this PR. From 50K to 1M records. 80% are `stop_and_search` events. Well done fixing this bug Ayyub!

Screen Shot 2022-05-18 at 10 07 15
 
But it does increase significantly the processing and push time now that there are so many records.

@huy-d-nguyen Are you using the `stop_and_search` on the site right now? If we could exclude them for now then processing and importing would be much faster.

**Comment by pckhoi** [comment]
@ayyubibrahimi feel free to merge this branch now.

**Pull request merged**